src/core/css/SearchResults.scss

```diff
@@ -2,7 +2,7 @@
 
 .SearchResults-list {
   margin: 0;
-  padding: 10px;
+  padding: 0;
```

Comment:
I'm guessing this sparked the change to `.App-content` since this got 20px of bottom padding then?